aleph-client/src/lib.rs

```diff
@@ -71,6 +71,97 @@ pub type Header = GenericHeader<BlockNumber, BlakeTwo256>;
 pub type KeyPair = sr25519::Pair;
 pub type Connection = Api<KeyPair, WsRpcClient>;
 
+/// 'Castability' to `Connection`.
+///
+/// Direct casting is often more handy than generic `.into()`. Justification: `Connection` objects
+/// are often passed to some macro like `compose_extrinsic!` and thus there is not enough
+/// information for type inferring required for `Into<Connection>`.
+pub trait AnyConnection: Clone {
+    fn as_con(&self) -> Connection;
```

Comment:
can we call it without abbreviations, `as_connection`

Reply:
renamed